v0/sources/appsflyer/transform.js

```diff
@@ -0,0 +1,50 @@
+const Message = require("../message");
+
+const { removeUndefinedAndNullValues } = require("../../util");
+
+function processEvent(event) {
+  const messageType = "track";
+
+  if (event.event_name) {
+    const eventName = event.event_name;
+    const message = new Message(`Appsflyer`);
```

Comment:
On the destination side, we name it as `AF`. I think we should use that here.

Reply:
done.